src/helpers.js

```diff
@@ -225,6 +225,10 @@ Helpers.guessSize = function (nInputs, nOutputs) {
   return (nInputs * 148 + nOutputs * 34 + 10);
 };
 
+Helpers.bytesToKb = function (bytes) {
```

Comment:
This function name is confusing (sorry, I know I suggested it). Converting bytes->kb would _divide_ by 1000. This is multiplying because it's actually converting sat/b->sat/kb.